langmeta/langmeta/shared/src/main/scala/org/langmeta/io/AbsolutePath.scala

```diff
@@ -29,6 +30,7 @@ sealed abstract case class AbsolutePath(toNIO: nio.Path) {
   def isFile: Boolean = FileIO.isFile(this)
   def isDirectory: Boolean = FileIO.isDirectory(this)
   def readAllBytes: Array[Byte] = FileIO.readAllBytes(this)
+  def write(proto: GeneratedMessage): Unit = FileIO.write(this, proto)
```

Comment:
Let's not add semanticdb specific methods to `AbsolutePath`. In fact, I regret adding `readAllBytes/isDirectory/isFile`, I think those should be static methods on some object like `nio.Files`. I would love to tear out org.langmeta.io into a standalone project one day so this method here is an odd fish.

Reply:
Done.

Reply:
Remove this method